### PR TITLE
ENCD-4215-enable-google-set

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

File: src/encoded/static/libs/jsonld.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        },
+    };
+
+    // set description
+    if (context['@type'].some(type => type.toLowerCase() === 'annotation')) {
+        mappedData.description = context.annotation_type;
+    } else if (context['@type'].some(type => type.toLowerCase() === 'experiment' || type.toLowerCase() === 'functionalcharacterizationexperiment')) {
+        mappedData.description = context.target_label ?
+        `[${context.target_label}] ${context.assay_title} ${context.biosample_summary}`.trim() :
+        `${context.assay_title} ${context.biosample_summary}`.trim();
```
Comment (forresttanaka) [inline src/encoded/static/libs/jsonld.js]: Is it OK that `biosample_summary` often doesn’t exist (https://test.encodedcc.org/search/?type=Experiment&biosample_summary!=*)? I see on http://localhost:6543/experiments/ENCSR000INT/ that the description property is “TF ChIP-seq undefined.”

Comment (zoldello) [reply src/encoded/static/libs/jsonld.js]: @forresttanaka Thanks. I added a condition to replace undefined with empty string. In line 121, I added the condition on target_label and realized it after I have pushed the commit; , but I assume that is okay, though..?

**Review by forresttanaka — DISMISSED** [review]

**Comment by zoldello** [comment]
@forresttanaka I made some changes in different PRs. Do you prefer I rebase and squash the commits?

**Comment by forresttanaka** [comment]
No, you can leave it this way. No merge conflicts.

File: src/encoded/static/libs/jsonld.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        keywords: [...context.internal_tags, ...[context.accession, context.uuid, 'ENCODE', 'ENCODED', 'DCC', 'Encyclopedia of DNA Elements']],
+        measurementTechnique,
+        includedInDataCatalog: {
+            '@type': 'DataCatalog',
+            name: 'ENCODE: Encyclopedia of DNA Elements',
+            url: `${baseUrl}`,
+        },
+    };
+
+    const partialDescription = [context.award.project, context.award.name, context.lab.title].filter(d => d !== '').join(' - ');
```
Comment (forresttanaka) [inline src/encoded/static/libs/jsonld.js:117]: I think technically you don’t need the filtering step, as award and lab are required in all three dataset types, and project and name are required in award, and title is required in lab.

**Review by forresttanaka — APPROVED** [review]
How well this complies with Google’s rules I don’t know, but the Javascript looks correct to me.

**Pull request merged**